cmd/kops/validate_cluster.go

```diff
@@ -94,37 +96,38 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		clientcmd.NewDefaultClientConfigLoadingRules(),
 		&clientcmd.ConfigOverrides{CurrentContext: contextName}).ClientConfig()
 	if err != nil {
-		return fmt.Errorf("cannot load kubecfg settings for %q: %v", contextName, err)
+		return fmt.Errorf("Cannot load kubecfg settings for %q: %v\n", contextName, err)
```

Comment:
> once go gets exceptions, of course

https://golang.org/doc/faq#exceptions

I don't think that will ever be happening - also I always fancied a codebase that the 1st level error message was capitalized, and 2nd+ tiers were lower case.. but I am a dreamer.. either way we can sneak it in later - I even think there is an issue for revisiting all of our error messages for uniformity somewhere